src/utils.js

```diff
@@ -1,30 +1,51 @@
+
 export const ORG_MAP = {
```

Comment:
NOTE: I'd like to propose using 3-4 chars for these IDs... not that there are going to be that many more provinces, etc... however having two had made it easier to have collisions... and having more letters might make it a little more guessible... again, not an issue as I can't see much changes, but typically ISO would be 3 chars minimum for most states/provinces, and countries as well... which is the 2/3

Reply:
provinces/territories are all 2 letters, I shouldn't have bothered using JO for Judo Ontario, should've just used ON. Too late now without migrating all the old values, but as you say, it's unlikely the number of organizations will grow

Reply:
I'll take your lead on how you want them changed. Just use the 2 character official provincial code?

Reply:
yea, let's go with that, should be enough for us.

Reply:
I've converted all province two-letter codes to official ones from Canada.